src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5245,4 +5245,41 @@ nuget update -Self</value>
   <data name="CommandMSBuildVersion" xml:space="preserve">
     <value>Specifies the version of MSBuild to be used with this command. Supported values are 4, 12, 14. By default the MSBuild in your path is picked, otherwise it defaults to the highest installed version of MSBuild.</value>
   </data>
+  <data name="AddCommandDescription" xml:space="preserve">
+    <value>Adds a given package to the specified source if it is a file source. http sources are not supported.</value>
```

Comment:
Discussed with @yishaigalatzer and updated the command description both for add and init. Check it out in the next iteration.